app/jobs/preassembly_job.rb

```diff
@@ -8,6 +8,6 @@ def perform(job_run)
     job_run.started
     bc = job_run.batch_context
     result = bc.batch.run_pre_assembly
-    result ? job_run.completed : job_run.errored
+    result ? job_run.completed : job_run.completed_with_errors
   end
```

Comment:
this is the case where at least one object causes an error, we go into our new state